system/API/ResponseTrait.php

```diff
@@ -95,7 +95,7 @@
 	 * Provides a single, simple method to return an API response, formatted
 	 * to match the requested format, with proper content-type and status code.
 	 *
-	 * @param null    $data
+	 * @param mixed    $data
```

Comment:
wouldn't line 330 be changed to match?

Reply:
Yes, I'm still working on fix all my IDE warning on system folder.

Reply:
Ok. I have added WIP to the PR title ... edit that out when you are done.